src/App.tsx

```diff
@@ -105,10 +103,10 @@ const App = () => {
 		flowId,
 		debug,
 		tenant: tenantId,
-		theme: flowTheme,
-		'style-name': styleName,
+		'style-id': styleName,
```

Comment:
This is calling the react-sdk so it will be `styleId`
https://github.com/descope/descope-js/pull/782/files